sdk/monitor/azure-monitor-query/tests/test_metrics_client_async.py

```diff
@@ -52,3 +52,13 @@ async def test_batch_metrics_granularity(self, recorded_test, monitor_info):
                 assert metric.timeseries
                 for t in metric.timeseries:
                     assert t.metadata_values is not None
+
+    @pytest.mark.asyncio
+    async def test_client_different_endpoint(self):
+        credential = self.get_credential(MetricsClient)
```

Comment:
Probably doesn't matter because we aren't getting a token, but should this have `is_async=True`?

Reply:
Ah, I'll throw that in for accuracy and consistency. Good catch!